gnomad/resources/grch38/gnomad.py

```diff
@@ -395,7 +395,7 @@ def coverage_tsv_path(data_type: str, version: Optional[str] = None) -> str:
     if data_type == "exomes":
         if version is None:
             version = CURRENT_EXOME_RELEASE
```

Comment:
Should this be updated to `CURRENT_EXOME_COVERAGE_RELEASE` to match genomes?

Reply:
yes, thank you!